db/migrate/20180619122743_add_rating_to_recipes.rb

```diff
@@ -0,0 +1,5 @@
+class AddRatingToRecipes < ActiveRecord::Migration[5.2]
+  def change
+    add_column :recipes, :rating, :integer
```

Comment:
Why would an average rating be an `integer` and not a `float`, we need at least one decimal to say, e.g.
Rating: 3.4 / 5

Reply:
We all talked about this and we all decided to keep it whole numbers the average will just rounded to the nearest whole number.  This will not be changed.